### PR TITLE
Fix/using new icons

### PR DESCRIPTION
**Pull request by ferreroltd**
This PR is meant to remove the old SystemIcon with the new product design icons, it impacts a little bit the appearance of some icons and a little bit the api of some interfaces, please verify

### CODE REVIEW

**Comment by ferreroltd** [comment]
Currently checking why the /faucet is not rendering the side navigation

**Review by KristinaSpasevska — CHANGES_REQUESTED** [review]
These icons stay with these colors?

Screenshot 2024-03-22 at 15 35 22
Screenshot 2024-03-22 at 15 36 37
Screenshot 2024-03-22 at 15 44 29

**Comment by isaozler** [comment]
> These icons stay with these colors?

The icons should have the following tokens assigned based on the feature/functionality. 

**Default Icons / Fallback**
`kda.foundation.color.icon.base.default`

**Brand Icons**
`kda.foundation.color.icon.brand.primary.default`
`kda.foundation.color.icon.brand.secondary.default`

**Semantic Icons**
`kda.foundation.color.icon.semantic.positive.default`
`kda.foundation.color.icon.semantic.negative.default`
`kda.foundation.color.icon.semantic.warning.default`
`kda.foundation.color.icon.semantic.info.default`

**Comment by isaozler** [comment]
@KristinaSpasevska does this ⬆️ help?

File: packages/apps/tools/src/components/Common/Layout/partials/Sidebar/styles.css.ts [hunk 1]
```diff
@@ -2,10 +2,21 @@ import { atoms, darkThemeClass, tokens } from '@kadena/react-ui/styles';
...
     position: 'relative',
+  }),
+]);
+
+export const iconWrapperClass = style([
+  atoms({
+    display: 'block',
+  }),
+  {
+    height: '16px',
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/components/Common/Layout/partials/Sidebar/styles.css.ts:18]: Can we get these values from tokens?

**Review by sanderlooijenga — APPROVED** [review]

**Review by KristinaSpasevska — APPROVED** [review]

**Pull request merged**